### PR TITLE
Changed the default timezone of monitor to UTC

### PR DESCRIPTION
**Pull request by ShlomiBalalis**
## PR pre-checks (self review)

~~- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)~~
~~- [ ] I gave variables/functions meaningful self-explanatory names~~
~~- [ ] I didn't leave commented-out/debugging code~~
~~- [ ] I didn't copy-paste code~~
- [x] I added the relevant `backport` labels
~~- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)~~
~~- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
~~- [ ] All new and existing unit tests passed (CI)~~
~~- [ ] I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

**Comment by amoskong** [comment]
Why not change data_dir/scylla-dash-per-server-nemesis.master.json?

File: data_dir/scylla-dash-per-server-nemesis.3.1.json [hunk 1]
```diff
@@ -73,7 +73,7 @@
             "3.1"
         ],
         "style": "dark",
-        "timezone": "browser",
+        "timezone": "utc",
```
Comment (amoskong) [inline data_dir/scylla-dash-per-server-nemesis.3.1.json]: Looks good, we always to change it manually for matching with the log time.

Comment (ShlomiBalalis) [reply data_dir/scylla-dash-per-server-nemesis.3.1.json]: Changed it in any other configuration file as well

**Comment by ShlomiBalalis** [comment]
@amoskong Changed it in any other configuration file as well

**Pull request merged**